CONTRIBUTING.md

```diff
@@ -70,7 +70,25 @@ $ git commit -m "Update javadoc for API."
 Continuous integration builds the project, runs the tests, and runs multiple
 types of static analysis.
 
-Before running the build init repository dependencies by running `make init-git-submodules`.
+##### Setup the project locally
+
+Fork and clone the project
+```
+git clone <projectpath>
+```
+From the root project directory, initialize repository dependencies 
+```
+make init-git-submodules
+```
+run this gradle build task
+```
+./gradlew opentelemetry-all:build
```

Comment:
Could we have a make target to build all instead of running Gradle directly? I don't like the following paragraph that build can fail and it would require a separate command to run.

We should make it as simple as possible to build.